Tools/ardupilotwaf/boards.py

```diff
@@ -176,7 +176,7 @@ def configure_env(self, cfg, env):
                 '-fcolor-diagnostics',
 
                 '-Wno-gnu-designator',
-                '-Wno-inconsistent-missing-override',
+                '-Werror=inconsistent-missing-override',
```

Comment:
Can we separate the forced error from the disabled warnings?

Reply:
As in re-order things?

Reply:
Assumed this was reordering things....